mailpoet/assets/js/src/newsletters/listings/standard.jsx

```diff
@@ -4,18 +4,14 @@ import MailPoet from 'mailpoet';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
 
-import confirmAlert from 'common/confirm_alert.jsx';
-import Tags from 'common/tag/tags';
-import Listing from 'listing/listing.jsx';
-import QueueStatus from 'newsletters/listings/queue_status.jsx';
-import Statistics from 'newsletters/listings/statistics.jsx';
-import {
-  addStatsCTAAction,
-  checkCronStatus,
-  checkMailerStatus,
-} from 'newsletters/listings/utils.jsx';
-import NewsletterTypes from 'newsletters/types';
-import { GlobalContext } from 'context/index.jsx';
+import confirmAlert from '../../common/confirm_alert.jsx';
+import Tags from '../../common/tag/tags';
+import Listing from '../../listing/listing.jsx';
+import QueueStatus from './queue_status.jsx';
+import Statistics from './statistics.jsx';
+import { addStatsCTAAction, checkCronStatus, checkMailerStatus } from './utils';
+import NewsletterTypes from '../types';
+import { GlobalContext } from '../../context';
```

Comment:
This is one of those instances that actually is not related to my goal but felt like `leave the file nicer than when you opened it`.
The size concern: I crossed checked the artifcats with the ones from master to make sure they are not growing ( actually public.js shrank ) and that's why I pointed out that we need a measure to watch build sizes so that we can add changes and not be too wary.
On the other hand there is another implicit decision involved about named exports and entry files and ... that we can discuss later on, the only reason for adding entry files here without discussion was to make it easier for the premium plugin to import them, I'm fine to revert back this change.